xformers/benchmarks/LRA/code/model_wrapper.py

```diff
@@ -182,29 +243,28 @@ def __init__(self, config, model_name):
 
     def forward(self, input_ids_0, input_ids_1, mask_0, mask_1, label):
```

Comment:
I know that it's not because of this PR, but would it be possible to add some comments here/above/below ? Basically this part of the code base is not of the same quality as the rest I believe (comments/typie hint/code coverage for instance), and I would take the opportunity of this PR to try to raise it little by little ?